torch/csrc/utils/device_lazy_init.h

```diff
@@ -28,7 +28,7 @@ void set_requires_device_init(at::DeviceType device_type, bool value);
 
 static inline void maybe_initialize_device(at::Device& device) {
   // Add more devices here to enable lazy initialization.
-  if (device.is_cuda() || device.is_xpu()) {
+  if (device.is_cuda() || device.is_xpu() || device.is_privateuseone()) {
```

Comment:
My concern here is that calling the function below assumes that a `._lazy_init()` method is defined on the device module.
I think it would be best to not have this requirement and if _lazy_init() is not defined, we always initialize.

Reply:
@albanD  Do you have an idea to handle both define `_lazy_init()` and not define `_lazy_init()`?
our device need to define lazy_init.

Reply:
please review my PR， thanks. @ezyang  @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @aakhundov @ColinPeppler @amjames @desertfire @chauhang